conf/pacific/upgrades/upgrade_small-cluster.yaml

```diff
@@ -0,0 +1,42 @@
+globals:
```

Comment:
This file may not be required, if we plan to newer `conf/pacific/upgrades/upgrades.yaml` config.

Reply:
I think it would be nice if we can keep a conf handy in case if any execution to be done on an upgraded cluster and to have a minimal cluster with lesser amount of time and resources.